src/renderer/component/fileCard/view.jsx

```diff
@@ -59,9 +56,20 @@ class FileCard extends React.PureComponent<Props> {
       obscureNsfw,
       claimIsMine,
       pending,
-      showPrice,
+      subscribed,
     } = this.props;
 
+    if (!claim) {
+      return (
+        <div className="card card--small">
+          <div className="card--placeholder card__media" />
+          <div className="card--placeholder placeholder__title" />
+          <div className="card--placeholder placeholder__channel" />
+          <div className="card--placeholder placeholder__date" />
```

Comment:
Should this be its own component? I'm not sure.

I think it could be helpful to see the classes that are used below to keep the layout consistent

Reply:
Doesn't contain a lot, do you expect to reuse this within other components?

Reply:
I see similar below, but it's not the same.